apps/src/turtle/turtle.js

```diff
@@ -403,6 +403,10 @@ class Visualization {
     this.heading = heading;
   }
 
+  pointToDirection(degreesRight) {
+    this.setHeading(this.heading + degreesRight);
+  }
```

Comment:
I'm not sure this does what we want.  Do we have test coverage for the 'PT' command, like we have for `artist.step('sticker', ...)`?

Reply:
In my attempt to split the PR's (creation of the blocks & code generation), I forgot to delete this portion of code.  I deleted the 'PT' command from artist.step.